### PR TITLE
Pin MarkupSafe to avoid import error

### PR DESCRIPTION
**Pull request by sthapa**
soft_unicode was removed from MarkupSafe in 2.1.0.  Pin to a working version.  See https://github.com/ssl-hep/ServiceX_App/runs/5326186779?check_suite_focus=true for a test run where this caused a failure

Signed-off-by: Suchandra Thapa <suchandra@gmail.com>

### CODE REVIEW

File: servicex/rabbit_adaptor.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=C0111,C0103,R0205
 import random
+import logging
```
Comment (BenGalewsky) [inline servicex/rabbit_adaptor.py:31]: Are these logging changes related to markupsafe?

Comment (sthapa) [reply servicex/rabbit_adaptor.py:31]: No, I'm not sure how it got in here since I created a new branch of develop.  I'll resubmit a new PR that's off a clean branch
